scripts/check-component-features.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I remember changing all of those to `#!/usr/bin/env bash` at #2458, seems like I missed some. Maybe we need to add a linter for this...

Reply:
Good news, Nix won't run them at all without using this, so we have a built in lint since the CI literally won't run those scripts. :)